pages/leave-policies.md

```diff
@@ -13,6 +13,11 @@ The maximum annual leave you can carry over from calendar year to calendar year
 
 Sick leave accrues at four hours per pay period for all federal employees, regardless of how long they've worked for the federal government. There is no maximum amount of sick leave you can accrue, and all unused hours carry over to the next calendar year. If you transfer from another federal agency, your sick leave transfers with you.
 
+### Unpaid leave
+
+Employees can request a supervisor for unpaid leave of less than four weeks at
```

Comment:
@pburkholder Thanks for proposing language! I suggest the following edits:
* Remove `a supervisor`, so --> `Employees can request unpaid leave`. (Both because it's the case for most of this and because they're not requesting someone to supervise their unpaid leave.)
* Prepend `Unlike leave without pay (see below),` to the next sentence, to give context for why someone would otherwise think they might have to have been employed for 12 months.